tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/store/AbstractDBStore.java

```diff
@@ -179,7 +179,7 @@ public AbstractDBStore(Configuration conf) throws InternalException {
           }
         }
      }
-    } catch (Exception se) {
+    } catch (Throwable se) {
```

Comment:
Looks good, but ```Throwable``` occurred while dropping base tables is consumed at Line 175. Please fix this too.